tabbycat/api/serializers.py

```diff
@@ -105,3 +118,77 @@ def update(self, instance, validated_data):
         else:
             self.instance.speaker_set.set(speakers)
         return self.instance
+
+
+class SpeakerSerializer(serializers.ModelSerializer):
+    url = SpeakerHyperlinkedIdentityField(view_name='api-speaker-detail')
+    categories = TournamentHyperlinkedRelatedField(
+        many=True,
+        view_name='api-speakercategory-detail',
+        queryset=SpeakerCategory.objects.all()
```

Comment:
We use trailing commas when the closing parenthesis/bracket/brace isn't on the same line as the last time (see [this section of PEP 8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)) 😎

Our bad for not including it in the flake8 checks 😳 I'll fix this and add the flake8 checks, I just thought I should give a heads-up for next time!